doc/classes/EditorSettings.xml

```diff
@@ -188,6 +188,9 @@
 			If [code]true[/code], automatically switches to the [b]Remote[/b] scene tree when running the project from the editor. If [code]false[/code], stays on the [b]Local[/b] scene tree when running the project from the editor.
 			[b]Warning:[/b] Enabling this setting can cause stuttering when running a project with a large amount of nodes (typically a few thousands of nodes or more), even if the editor window isn't focused. This is due to the remote scene tree being updated every second regardless of whether the editor is focused.
 		</member>
+		<member name="debugger/auto_switch_to_stack_trace" type="bool" setter="" getter="">
+			If [code]true[/code], automatically switches to the [b]Stack Trace[/b] panel when the debugger hits a breakpoint or steps.
+		</member>
```

Comment:
```suggestion
		<member name="debugger/auto_switch_to_stack_trace" type="bool" setter="" getter="">
			If [code]true[/code], automatically switches to the [b]Stack Trace[/b] panel when the debugger hits a breakpoint or steps.
			[b]Note:[/b] This option won't do anything if the bottom panel switching is locked using the pin button in the corner of the bottom panel.
		</member>
```

Reply:
I don't think that's the case. What the pin locks are the bottom's dock context switching while this setting is about the tab switching inside the Debugger context.

Reply:
Ah, apologies, I misinterpreted the PR.